openslides/motions/static/js/motions/site.js

```diff
@@ -285,7 +285,23 @@ angular.module('OpenSlidesApp.motions.site', [
     'gettextCatalog',
     'Editor',
     'Config',
-    function(gettextCatalog, Editor, Config) {
+    'formlyConfig',
+    function(gettextCatalog, Editor, Config, formlyConfig) {
+        formlyConfig.setType({
```

Comment:
Is this something generic? It looks so, so I propose to move it to a reusable factory or other service.